### ISSUE TITLE
Cannot dynamically update placeholder options

### ISSUE THREAD

**Issue by mrowles** [jspaine/quill-placeholder-module#7]
I've tried to emulate [this possible solution here](https://stackoverflow.com/questions/45796434/quilljs-reload-toolbar-controls) in order to dynamically update placeholder options, but the click functionality seems to disappear when you add a new one. I think it's because these are registered when the module is created. Would be good to get this working dynamically.

**Comment by jspaine** [comment]
What are you trying to do, add/remove placeholders?

**Comment by mrowles** [comment]
Yeah at a basic level. I understand I could hold off on loading the Quill instance until I have them loaded, say, from a HTTP request, but I also have the use case of having users being able to add them in real-time. Quill has this annoying limitation where we cannot re-instantiate the editor or even the toolbar. `destroy()` was removed in a previous version.

**Comment by jspaine** [comment]
Ok I'll see if there's anything I can do in here. Think you might just have to throw away the editor, create a new one and copy the state though.

**Comment by massive688** [comment]
> Ok I'll see if there's anything I can do in here. Think you might just have to throw away the editor, create a new one and copy the state though.

12313

**Comment by Dhavalc2012** [comment]
@mrowles Have you tried to change the key of the component? It will refresh the placeholders which worked for my usecase.
